app/src/main/java/View/MainScreen.java

```diff
@@ -0,0 +1,41 @@
+/*
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
+ * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
+ */
+package View;
+
+import de.gurkenlabs.litiengine.Game;
+import de.gurkenlabs.litiengine.gui.screens.GameScreen;
+import java.awt.Color;
+import java.awt.Graphics2D;
+import java.awt.image.BufferedImage;
+import java.io.File;
+import javax.imageio.ImageIO;
+
+/**
+ *
+ * @author andru
+ */
+public class MainScreen extends GameScreen {
+    
+    public MainScreen() {
+        super("TEST");
+    }
+
+    @Override
+    public void render(final Graphics2D g) {
+        super.render(g);
+        g.setColor(Color.RED);
+        Game.graphics().renderText(g, "Test text", 100, 100);
+        
+        
+        BufferedImage img = null;

```

Comment:
I recommend not opening an image at all, because we don't have any images stored in the repo. Also this code opens the image EVERY FRAME!